FaceDetectionServer/moedx/tracker/utils.py

```diff
@@ -0,0 +1,80 @@
+import psutil
+import logging
+import os
+import json
+from subprocess import Popen, PIPE
+
+logger = logging.getLogger(__name__)
+
+def usage_gpu():
+    """
+    Execute the "nvidia-smi" command and parse the output to get the GPU usage.
+    """
+    nvidia_smi = "nvidia-smi"
+    try:
+        p = Popen([nvidia_smi,"-q", "-d", "UTILIZATION"], stdout=PIPE)
+    except FileNotFoundError:
+        logger.error("%s not found. GPU usage not supported." %nvidia_smi)
```

Comment:
Should be part of the driver package (at least, on linux). "GPU usage reporting not supported."

Reply:
I'll add your suggestion to the error log. Just pointing out here that this same code is run on both GPU and non-GPU servers. The error is really for the non-GPU case. What is returned by the /server/usage/ endpoint has a different set of fields for each case (that is, the GPU related ones are missing in the non-GPU case).